src/main/java/org/example/constant/UrlPattern.java

```diff
@@ -0,0 +1,34 @@
+package org.example.constant;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum UrlPattern {
+    HTML("/html"), JSON("/json"), HELLO("/hello");
+
+    private static final String WAR_NAME = "/DeveloperApi";
+
+    private final String url;
+
+    private static final Map<String, UrlPattern> URL_PATTERN_STRING_MAP = new HashMap<>();
+
+    static {
+        URL_PATTERN_STRING_MAP.put(HELLO.getUrl(), HELLO);
+        URL_PATTERN_STRING_MAP.put(JSON.getUrl(), JSON);
+        URL_PATTERN_STRING_MAP.put(HTML.getUrl(), HTML);
+    }
+
+    UrlPattern(String url) {
+        this.url = url;
+    }
+
+    public String getUrl() {
+        String finalUrl = "%s%s";
+        return String.format(finalUrl, WAR_NAME, url);
```

Comment:
It is not ok, to change the initial value of the property that user tries to get

Reply:
fixed